localstack/services/awslambda/lambda_executors.py

```diff
@@ -362,7 +362,8 @@ def prime_docker_container(self, runtime, func_arn, env_vars, lambda_cwd):
 
                 mount_volume = not config.LAMBDA_REMOTE_DOCKER
                 lambda_cwd_on_host = Util.get_host_path_for_path_in_docker(lambda_cwd)
-                lambda_cwd_on_host = Util.format_windows_path(lambda_cwd)
+                if (':' in lambda_cwd and '\\' in lambda_cwd):
+                    lambda_cwd_on_host = Util.format_windows_path(lambda_cwd)
```

Comment:
I think we should change this to (looks like this was a bug, even prior to your PR):
```
if (':' in lambda_cwd and '\\' in lambda_cwd):
    lambda_cwd_on_host = Util.format_windows_path(lambda_cwd_on_host)
```